src/end_entity.rs

```diff
@@ -116,17 +118,19 @@ impl<'a> EndEntityCert<'a> {
     /// time).
     pub fn verify_is_valid_tls_client_cert(
         &self,
-        supported_sig_algs: &[&SignatureAlgorithm],
-        &TlsClientTrustAnchors(trust_anchors): &TlsClientTrustAnchors,
-        intermediate_certs: &[&[u8]],
+        supported_sig_algs: &'a [&'a SignatureAlgorithm],
```

Comment:
It is not obvious to me why we need the lifetime annotations here, given that we aren't returning any lifetime-bound data?

Reply:
I'll take a look. I suspect its an artifact of some earlier iteration that I didn't clean up correctly.

Reply:
Nope, I guess it's fine.